openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -966,6 +971,65 @@ def test_multiple_published_course_runs(self):
 
         self.assertTrue(data['is_learner_eligible_for_one_click_purchase'])
 
```

Comment:
Please add a test for 'no-id-professional' or update if there is one already available.

Reply:
From what I can tell, by the time the data gets here about products, any instance of no-id-professional is converted to professional, so I'm not entirely convince we need to test for it here.